src/rules/at-if-no-null/__tests__/index.js

```diff
@@ -0,0 +1,41 @@
+import rule, { ruleName, messages } from "..";
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  syntax: "scss",
+
+  accept: [
+    {
+      code: `a {
+      @if ($x) {}
+    }`,
+      description: "does not use the != null format"
+    },
+    {
+      code: `a {
+      @if not ($x) {}
+    }`,
+      description: "does not use the == null format"
+    }
+  ],
```

Comment:
There could be some more tests with calculations and other conditions to make sure that there are no false positives/negatives.

Reply:
@nex3 any ideas on good tests or language constructs that users might be using in these if/else statements?

Reply:
When I'm looking for examples, I often look through big open-source corpuses like https://github.com/angular/components/tree/master/src/material or https://github.com/carbon-design-system/carbon/tree/master/packages.

Reply:
Great idea! I didn't know of any large Sass codebases.

I grepped through carbon-design and came across a common pattern (if $x != null and expr). I made adjustments to the rule for that pattern.